data/util/NumericShaperUtility.js

```diff
@@ -0,0 +1,294 @@
+define(
```

Comment:
There is nothing special in data/util. I had to select a place to put NumericShaperUtility.js in, so I selected data/util. Do you recommend cldr ?